package.json

```diff
@@ -230,10 +230,20 @@
   },
   "browserslist": {
     "production": [
-      "defaults"
+      "last 2 Chrome versions",
+      "last 2 Firefox versions",
+      "last 2 Edge major versions",
+      "last 2 Safari major versions",
+      "last 3 iOS major versions",
+      "Firefox ESR"
     ],
     "development": [
-      "defaults"
+      "last 2 Chrome versions",
+      "last 2 Firefox versions",
+      "last 2 Edge major versions",
+      "last 2 Safari major versions",
+      "last 3 iOS major versions",
+      "Firefox ESR"
```

Comment:
I think these constraints are good for development.

Reply:
i think i wanted to use the same values as production, i know some people do not.